llvm/tools/llvm-exegesis/llvm-exegesis.cpp

```diff
@@ -107,6 +107,12 @@ static cl::opt<Benchmark::RepetitionModeE> RepetitionMode(
                    "Middle half loop mode")),
     cl::init(Benchmark::Duplicate));
 
+static cl::opt<bool> AArch64DisablePacControl(
```

Comment:
This should be moved to where it is actually used.

Reply:
Yes added in the Target.cpp, thanks